### PR TITLE
[terra-hookshot] Change variable name to fix resize popup issue.

### LINKED ISSUE #1344: [terra-hookshot] Popup throwing error on increasing width
# Bug Report

## Description
Consumers are facing errors on increasing the popup width after certain point, as a couple of variables were overlooked in hookshot during refactoring.

## Additional Context / Screenshots

image

## Expected Behavior
It should not throw any errors.

## Possible Solution
Rename the variables overlooked during refactoring.

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Change Variable name to fix resize popup issue.
Closes #1344 

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-hookshot/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Fixed
+ * Change variable names overlooked in refactoring to fix resize popup issue
```
Comment (supreethmr) [inline packages/terra-hookshot/CHANGELOG.md:5]: Formatting for change log heading seems to be missing here..!
```suggestion
### Fixed
* Fixed error thrown on resizing the popup due to incorrect variables name.
```

**Review by supreethmr — APPROVED** [review]

**Review by dkasper-was-taken — APPROVED** [review]

**Review by StephenEsser — APPROVED** [review]

**Comment by dkasper-was-taken** [comment]
Might be worthwhile to add a jest test for this function if one isn't present.

**Review by mjhenkes — CHANGES_REQUESTED** [review]
Please add a jest test covering this case.

**Review by pranav300 — APPROVED** [review]

**Comment by saket2403** [comment]
Added jest 59c6d50

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**